flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/autoscaler/ScalingExecutor.java

```diff
@@ -99,14 +103,27 @@ public boolean scaleResource(
             return false;
         }
 
+        var scalingReport = scalingReport(scalingSummaries);
+        eventRecorder.triggerEvent(
+                resource,
+                EventRecorder.Type.Normal,
+                EventRecorder.Reason.ScalingReport,
+                EventRecorder.Component.Operator,
+                scalingReport);
+
+        if (!conf.get(SCALING_ENABLED)) {
+            return false;
+        }
```

Comment:
I think that is completely expected. This will make events useful in case the user prefers to trigger actions manually. For instance when trying out the autoscaler initially.

When scaling actions are disabled events are actually actionable to the user this way

Reply:
Absolutely, I think this is a great addition, but only if we inform the users that the scaling wasn’t actually executed. We can include this in the event message. Otherwise I foresee confusion around whether the scaling was executed or not.

Reply:
I hear you, how about this:
```
2023-01-25 13:16:04,184 o.a.f.k.o.u.EventCollector     [INFO ] >>> Event  | Info    | SCALINGREPORT   | Scaling vertices: Vertex ID 611fc81374e72d3d19e2d14196df735c | Parallelism 1 -> 2 | Processing capacity 100,00 -> 157,00 | Target data rate 110,00
```
vs

```
2023-01-25 13:16:04,274 o.a.f.k.o.u.EventCollector     [INFO ] >>> Event  | Info    | SCALINGREPORT   | Recommended parallelism change: Vertex ID b6844a4ed9bebdbbd33e4ebc1662745e | Parallelism 1 -> 2 | Processing capacity 100,00 -> 157,00 | Target data rate 110,00
```

Reply:
That is awesome! :)